packages/dashboard/src/components/internalDashboard/index.tsx

```diff
@@ -370,7 +371,11 @@ const InternalDashboard: React.FC<InternalDashboardProps> = ({ messageOverrides
       </div>
       <div className="iot-dashboard-panes-area">
         <ResizablePanes
-          leftPane={<div className="dummy-content">Resource explorer pane</div>}
+          leftPane={
+            <div className="iot-resource-explorer-pane">
+              {query && <IotResourceExplorer query={query.assetTree.fromRoot()} />}
```

Comment:
should we have an empty state here if no query is provided?

Reply:
Yes, we probably should. Also, I need change the name of that prop from `query` to `assetTree` or something; that's a holdover from the original resource explorer which called the asset tree prop 'query' as well, which is really confusing because the query is something different. Thanks for the reminder :)